Sources/CryptoSwift/PKCS/PKCS7Padding.swift

```diff
@@ -29,13 +29,8 @@ struct PKCS7Padding: PaddingProtocol {
   func add(to bytes: Array<UInt8>, blockSize: Int) -> Array<UInt8> {
     let padding = UInt8(blockSize - (bytes.count % blockSize))
     var withPadding = bytes
-    if padding == 0 {
-      // If the original data is a multiple of N bytes, then an extra block of bytes with value N is added.
-      withPadding += Array<UInt8>(repeating: UInt8(blockSize), count: Int(blockSize))
-    } else {
-      // The value of each added byte is the number of bytes that are added
-      withPadding += Array<UInt8>(repeating: padding, count: Int(padding))
-    }
+    // The value of each added byte is the number of bytes that are added
+    withPadding += Array<UInt8>(repeating: padding, count: Int(padding))
```

Comment:
You're right, it cannot be zero since `a % b` is always `< b`.
But in that case you can event write it as `return bytes + Array...` to remove two extra lines (and maybe use less memory, but I'm not sure it changes anything on that)

Reply:
applied your suggestion